android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java

```diff
@@ -529,6 +529,12 @@ private boolean removeOrAnimateExitRecursive(View view, boolean shouldRemove) {
     }
 
     if (hasAnimatedChildren) {
+      if (tag == -1) {
+        // View tags are used to identify views, but it is hard for no-react views with -1 tag.
```

Comment:
I'm not sure if I understand "it is hard for no-react views with -1 tag" correctly, what exactly do you mean?

Reply:
I've accidentally merged this PR without fixing this comment 😱 
I opened this: https://github.com/software-mansion/react-native-reanimated/pull/4634 to fix it